os_stub/platform_lib/time_win.c

```diff
@@ -19,6 +19,13 @@ void libspdm_sleep_in_us(uint64_t microseconds)
 {
     uint64_t milliseconds;
 
+    /* If 0 is given as the sleep time interval, the thread will relinquish the remainder of its time slice.
+     * https://learn.microsoft.com/en-us/windows/win32/api/synchapi/nf-synchapi-sleep
+     * Return immediately to avoid unexpected re-scheduling. */
+    if (microseconds == 0) {
```

Comment:
Do we need change Linux version as well?

https://github.com/DMTF/libspdm/blob/main/os_stub/platform_lib/time_linux.c

Reply:
We're good on linux implementation since man page of `select()` says it will return immediately.

https://man7.org/linux/man-pages/man2/select.2.html
**Arguments** -> *timeout*
> If both fields of the timeval structure are zero, then select() returns immediately.